meta-lxatac-software/recipes-support/bcu/bcu_%.bbappend

```diff
@@ -1,4 +1,4 @@
-FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
+PV = "1.1.120"
```

Comment:
Maybe make the bump version more visible in the commit message. The rest is a side effect that should be noted.

I couldn't really figure out why a more recent bcu version is required anyway.

Reply:
I have added a bit about why I carry a `bcu_%.bbappend` to the commit message.
It is because I was the one who upstreamed the bcu recipe and I would like to test updates in meta-lxatac before sending them upstream.